config/routes.rb

```diff
@@ -29,13 +29,19 @@
     end
   end
 
-  namespace :api do
+  namespace :api, defaults: { format: "json" } do
     resources :school_search, only: %i[index]
     resource :notify_callback, only: :create, path: "notify-callback"
 
     namespace :v1 do
-      unless Rails.env.staging? || Rails.env.production?
-        resources :users, only: :index
+      namespace :participants do
+        resources :induction_progress, only: %i[create], path: "induction-progress"
+      end
+    end
+
+    namespace :public do
+      namespace :v1 do
+        resources :users, only: :index unless Rails.env.staging? || Rails.env.production?
       end
```

Comment:
As mentioned above, I really don't think `users` api will end up as `public` - I don't think returning everyone's emails and full names should be in that namespace. We will add auth to it to make it non-public before we release to staging and live.

Reply:
Then you can move to to non-public before releasing to staging and live, simple

Reply:
Look, users api is a work-in-progress, which is why it only lives in non-real environments. I chose not to include authentication in the API in the first place because I plan to reuse a lot of your work for it, and because E&L doesn't have a proper live environment yet.

Moving the api between namespaces makes our life harder - we want to iterate and improve it, but if it changes namespace while we are doing it, we will have to re-write all of our work. We know where it will eventually end up, so why would we move it around before releasing it?

And on a completely different note: If you strongly feel it should live elsewhere, that change should live in its own PR. This one already has enough changes in it.

Reply:
Ok, how about if we move it back to non-public namespace and disable authentication with a non-live flag, so all swagger docs and rspec tests will pass?